src/Tribe/Commerce/PayPal/Order.php

```diff
@@ -303,25 +312,26 @@ public function hydrate_from_transaction_data( array $transaction_data ) {
 	 *
 	 * @since 4.7
 	 *
-	 * @param array $criteria  {
+	 * @param array      $args   {
 	 *                         Optional. Arguments to retrieve orders. See WP_Query::parse_query() for all
 	 *                         available arguments.
 	 *
 	 * @type int    $post_id   ID, or array of IDs, of the post(s) Orders should be related to.
 	 * @type int    $ticket_id ID, or array of IDs, of the ticket(s) Orders should be related to.
 	 * }
+	 * @param null|array $fields List of fields to hydrate, or null for all.
 	 *
 	 * @return Tribe__Tickets__Commerce__PayPal__Order[]
 	 */
-	public static function find_by( array $args = array() ) {
-		$args = wp_parse_args( $args, array(
+	public static function find_by( array $args = [], $fields = null ) {
+		$args = wp_parse_args( $args, [
 			'post_type'   => Tribe__Tickets__Commerce__PayPal__Main::ORDER_OBJECT,
 			'post_status' => 'any',
 			'meta_key'    => self::$meta_prefix . 'payment_date',
 			'meta_type'   => 'DATETIME',
 			'order'       => 'DESC',
 			'orderby'     => 'meta_value',
-		) );
+		] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: f77b3641a072e5af5cc28c33ebfb24ea</sub>